src/Maker/MakeEntity.php

```diff
@@ -43,12 +43,14 @@
     private $fileManager;
     private $doctrineHelper;
     private $projectDirectory;
+    private $entityNamespace;
 
-    public function __construct(FileManager $fileManager, string $projectDirectory, DoctrineHelper $doctrineHelper)
+    public function __construct(FileManager $fileManager, DoctrineHelper $doctrineHelper, string $projectDirectory, string $entityNamespace)
```

Comment:
To avoid needing to inject the `$entityNamespace` ALSO here, let's add a `getEntityNamespace()` method to `DoctrineHelper`.